migrator/src/main/java/uk/nhs/digital/ps/migrator/model/hippo/TaxonomyLookup.java

```diff
@@ -0,0 +1,64 @@
+package uk.nhs.digital.ps.migrator.model.hippo;
+
+import java.io.*;
+import java.util.*;
+import org.yaml.snakeyaml.Yaml;
+
+public class TaxonomyLookup {
+
+    private final ArrayList<String> taxonomyKeys = new ArrayList<>();
+    private final static String HIPPO_TAXONOMY_KEY = "hippotaxonomy:key";
+
+    public TaxonomyLookup()
+    {
+        setup();
+    }
+
+    private void setup() {
+
+        // Load taxonomy yaml from file
+        // Currently uses development taxonomy file, but upon migration will use
+        // the full taxonomy structure from Initial Import of Taxonomy
+        Yaml yaml = new Yaml();
+        File taxonomyYamlFile = new File("./repository-data/development/src/main/resources/hcm-content/content/taxonomies/publication_taxonomy.yaml");
+
+        HashMap<String, String> map = null;
+
+        try {
+            map = yaml.load(new FileInputStream(taxonomyYamlFile));
+
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```

Comment:
I'd re-throw this as a RuntimeException so the application doesn't continue if the error occurs